### ISSUE TITLE
minidlna 1.1.5-4 can't load libraries

### ISSUE THREAD

**Issue by N0News** [Optware/Optware-ng#218]
Hello, after upgrading minidlna to version 1.1.5-4 i have this errors:
```
/opt/sbin/minidlnad: can't load library 'libpthread.so.1'
/opt/sbin/minidlnad: can't load library 'libdl.so.1'
```

I use buildroot-armv5eabi-ng-legacy on Synology DS212J. Manually moving this libraries from old uclibc-opt_1.0.15-2_arm.ipk helped, but please fix this.

**Comment by alllexx88** [comment]
Hi,
I've fixed few `libpthread` issues (`ffmpeg` and `liblzma0` packages), and now `minidlnad` works fine for me. I haven't experienced `libdl.so.1` dependency issue, so let's first upgrade all packages:
```
ipkg update
ipkg upgrade
```
And second use slightly modified `lddtree` (will later add it as a package) to see what binary was linked against `libdl`
```
ipkg install bash sed binutils
wget https://github.com/Optware/Optware-ng/files/1384849/lddtree.txt -o /opt/bin/lddtree
chmod +x /opt/bin/lddtree
/opt/bin/lddtree /opt/sbin/minidlnad
```
and send me output of the last command please

Thanks

**Comment by N0News** [comment]
OK, all done and things go bad for your version of lddtree:
`
/opt/bin/lddtree /opt/sbin/minidlnad
minidlnad => /opt/sbin/minidlnad/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
 (interpreter => none)
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
    libjpeg.so.8 => /lib/libjpeg.so.8
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
        libgcc_s.so.1 => /lib/libgcc_s.so.1
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
        libc.so.6 => /lib/libc.so.6
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
            ld-linux.so.3 => /lib/ld-linux.so.3
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
    libpng12.so.0 => /opt/bin/lddtree: line 204: cd: libpng12.so.0: No such file or directory
not found
/opt/bin/objdump: 'libpng12.so.0': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
    libid3tag.so.0 => /opt/bin/lddtree: line 204: cd: libid3tag.so.0: No such file or directory
not found
/opt/bin/objdump: 'libid3tag.so.0': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
    libsqlite3.so.0 => /lib/libsqlite3.so.0
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
        libdl.so.2 => /lib/libdl.so.2
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
        libpthread.so.0 => /lib/libpthread.so.0
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libavformat.so.56 => /opt/bin/lddtree: line 204: cd: libavformat.so.56: No such file or directory
not found
/opt/bin/objdump: 'libavformat.so.56': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libavutil.so.54 => /opt/bin/lddtree: line 204: cd: libavutil.so.54: No such file or directory
not found
/opt/bin/objdump: 'libavutil.so.54': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
    libexif.so.12 => /lib/libexif.so.12
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
        libm.so.6 => /lib/libm.so.6
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libiconv.so.2 => /opt/bin/lddtree: line 204: cd: libiconv.so.2: No such file or directory
not found
/opt/bin/objdump: 'libiconv.so.2': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libFLAC.so.8 => /opt/bin/lddtree: line 204: cd: libFLAC.so.8: No such file or directory
not found
/opt/bin/objdump: 'libFLAC.so.8': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libogg.so.0 => /opt/bin/lddtree: line 204: cd: libogg.so.0: No such file or directory
not found
/opt/bin/objdump: 'libogg.so.0': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libvorbis.so.0 => /opt/bin/lddtree: line 204: cd: libvorbis.so.0: No such file or directory
not found
/opt/bin/objdump: 'libvorbis.so.0': No such file
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/gnu-sed: can't load library 'libacl.so.1'
/opt/bin/objdump: '': No such file
    libc.so.0 => /opt/bin/lddtree: line 204: cd: libc.so.0: No such file or directory
not found
/opt/bin/objdump: 'libc.so.0': No such file
`

But I have installed ldd from repository:
`
ldd /opt/sbin/minidlnad
        libjpeg.so.8 => /opt/lib/libjpeg.so.8 (0x40017000)
        libpng12.so.0 => /opt/lib/libpng12.so.0 (0x4005b000)
        libid3tag.so.0 => /opt/lib/libid3tag.so.0 (0x4008c000)
        libsqlite3.so.0 => /opt/lib/libsqlite3.so.0 (0x400ab000)
        libavformat.so.56 => /opt/lib/libavformat.so.56 (0x40157000)
        libavutil.so.54 => /opt/lib/libavutil.so.54 (0x402ff000)
        libexif.so.12 => /opt/lib/libexif.so.12 (0x4037c000)
        libiconv.so.2 => /opt/lib/libiconv.so.2 (0x403be000)
        libFLAC.so.8 => /opt/lib/libFLAC.so.8 (0x404ab000)
        libogg.so.0 => /opt/lib/libogg.so.0 (0x4050c000)
        libvorbis.so.0 => /opt/lib/libvorbis.so.0 (0x40520000)
        libc.so.0 => /opt/lib/libc.so.0 (0x4055b000)
        libz.so.1 => /opt/lib/libz.so.1 (0x40637000)
        libass.so.9 => /opt/lib/libass.so.9 (0x4065c000)
        libavcodec.so.56 => /opt/lib/libavcodec.so.56 (0x40692000)
        libssl.so.1.0.0 => /opt/lib/libssl.so.1.0.0 (0x41597000)
        libcrypto.so.1.0.0 => /opt/lib/libcrypto.so.1.0.0 (0x415fc000)
        libbz2.so.1.0 => /opt/lib/libbz2.so.1.0 (0x41790000)
        ld-uClibc.so.1 => /opt/lib/ld-uClibc.so.1 (0x40000000)
        libfribidi.so.0 => /opt/lib/libfribidi.so.0 (0x417b0000)
        libglib-2.0.so.0 => /opt/lib/libglib-2.0.so.0 (0x417d6000)
        libintl.so.8 => /opt/lib/libintl.so.8 (0x418ff000)
        libfontconfig.so.1 => /opt/lib/libfontconfig.so.1 (0x41918000)
        libfreetype.so.6 => /opt/lib/libfreetype.so.6 (0x41961000)
        libswresample.so.1 => /opt/lib/libswresample.so.1 (0x419f2000)
        libx264.so.142 => /opt/lib/libx264.so.142 (0x41a16000)
        libvorbisenc.so.2 => /opt/lib/libvorbisenc.so.2 (0x41b4f000)
        libmp3lame.so.0 => /opt/lib/libmp3lame.so.0 (0x41be3000)
        libfdk-aac.so.1 => /opt/lib/libfdk-aac.so.1 (0x41c85000)
        liblzma.so.5 => /opt/lib/liblzma.so.5 (0x41d2a000)
        libexpat.so.1 => /opt/lib/libexpat.so.1 (0x41d5a000)
        libsoxr.so.0 => /opt/lib/libsoxr.so.0 (0x41d8a000)
        libgcc_s.so.1 => /opt/lib/libgcc_s.so.1 (0x41df7000)
`
Seems like minidlnad can work now.

**Comment by alllexx88** [comment]
> Seems like minidlnad can work now.

Great.

Thanks for the notice on `sed` too: fixed in 4b96b7b4d773914d8e02ee835ea312169058ccc5
